tintinweb/scapy-ssl_tls
# integration testsuite

created a basic integration testsuite
- manages external tls server (via Popen)
- implements python tls server (via multiprocessing.Process)
- implements example `full_rsa_connection_with_application_data.py`
- based on `unittest` to have a common test interface 
- moved to a different folder to separate the tests from unittests

todo:
- add tests for sniffer and other examples
- move common stuff to own module to have a clean testsuite
- fix bugs
- add external tests (check tls compatibility with public webservices)

Note: this is based on master without changes introduced with #77
